core/foundation/src/StringUtils.cxx

```diff
@@ -37,4 +38,21 @@ std::vector<std::string> Split(std::string_view str, std::string_view delims, bo
    return out;
 }
 
+
+/**
+ * \brief Concatenate vector of std::string
+ * \param[in] sep separator inbetween the strings
+ * \param[in] strings vector of strings
+ * \return the sep-delimited concatenation of strings
+ * \see https://stackoverflow.com/questions/9277906/stdvector-to-string-with-custom-delimiter
```

Comment:
```suggestion
```
I don't think it's necessary to quote stackoverflow in the ROOT reference guide :)